data/issues.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import active_newcomers
+from data.models import Contributor
+from data.webservices import webservices_url
+from data.convert import convert
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    hoster = hoster.lower()
+    if hoster == 'github':
+        IMPORT_URL = webservices_url('issues/github/all')
+    elif hoster == 'gitlab':
+        IMPORT_URL = webservices_url('issues/gitlab/all')
+
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+    issues = response.json()
+
+    # Removing issues which are not opened by newcomers
+    _issues = []
```

Comment:
`_` prefix denotes private.

a local variable is already private, so do not use `_` prefixes for local variables.

create a proper variable name which distinguishes this from `issues` above.